WordPress/gutenberg
# [RNMobile] Some tweaks in block picker

This PR introduces some changes in picker / action sheet such as:

* Options no more have block name in label
* There is a title with block name above options
* Fix a11y

Ref to gb-mobile: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2438

## Description

* Options no more have block name in label
* There is a title above options
* Fix a11y

## How has this been tested?




1. Open mobile app
2. Add `Buttons`
3. Open action sheet via ellipsis button
4. Check if all the options within picker are working properly
5. Check if the header contains block name

## Screenshots 

android | ios
--- | ---
<img width="427" alt="Screenshot 2020-06-08 at 15 50 22" src="https://user-images.githubusercontent.com/22746080/84038139-d401bf80-a99f-11ea-908a-305428331225.png"> | <img width="493" alt="Screenshot 2020-06-08 at 15 54 12" src="https://user-images.githubusercontent.com/22746080/84038507-54282500-a9a0-11ea-81c9-8444758e3899.png">

## Types of changes





## Checklist:
